assets/src/edit-story/units/dimensions.js

```diff
@@ -116,9 +121,15 @@ export function getBox(
   const displayFull = isFill || isBackground;
   return {
     x: dataToEditorX(displayFull ? 0 : x, pageWidth),
-    y: dataToEditorY(displayFull ? 0 : y, pageHeight),
+    y: dataToEditorY(
+      displayFull ? -(PAGE_WIDTH * PREVIEW_RATIO - PAGE_HEIGHT) / 2 : y,
+      pageHeight
+    ),
     width: dataToEditorX(displayFull ? PAGE_WIDTH : width, pageWidth),
-    height: dataToEditorY(displayFull ? PAGE_HEIGHT : height, pageHeight),
+    height: dataToEditorY(
+      displayFull ? PAGE_WIDTH * PREVIEW_RATIO : height,
+      pageHeight
+    ),
```

Comment:
I don't like this... better ideas?

Reply:
I don't like this either. But IMHO we can start here. The main reason for these is to ensure that image and video editors with pan/zoom continue to work as before. We might find another way to special-case them, but for now I think this is ok.